node_modules/oae-core/activity/js/activity.js

```diff
@@ -42,7 +42,8 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
                     'additionalClasses': 'col-xs-12 col-lg-6'
                 },
                 'displayOptionsSmall': {
-                    'additionalClasses': 'col-xs-6 col-sm-4 col-md-3 col-lg-2'
+                    'additionalClasses': (widgetData.context.resourceType === 'group') ?
+                        'col-xs-4 col-sm-3 col-md-4 col-lg-3' : 'col-xs-6 col-sm-4 col-md-3 col-lg-2'
```

Comment:
Ideally we have a better strategy for this. I'm not sure if I should just allow these col classes to come in as the activity widget settings, or if we should have named orientations. I'm leaning towards widget settings, in which case we probably want one for the `displayOptionsSmall` and the `displayOptionsLarge`